### PR TITLE
Add runtime changes to release note

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
$Subject

### CODE REVIEW

File: release-notes/swan-lake-alpha4-release-note.md [hunk 1]
```diff
@@ -66,7 +66,56 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+-Ckey=value
+```
+
+Key syntax:
+
+```
+key:= [[orgName .] packageName.moduleName .] variable
+```
+
+Command line arguments are supported for config variables with boolean , int , float , decimal , string and xml types.
```
Comment (HindujaB) [inline release-notes/swan-lake-alpha4-release-note.md:83]:
```suggestion
Command line arguments are supported for configurable variables with boolean , int , float , decimal , string and xml types.
```

File: release-notes/swan-lake-alpha4-release-note.md [hunk 2]
```diff
@@ -66,7 +66,56 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+
+Example usages
+
+```ballerina
+configurable int port = ?;
+```
+
+If configurable variable defined in the default module.
```
Comment (HindujaB) [inline release-notes/swan-lake-alpha4-release-note.md:91]: Do we need to include the single bal file case here?

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-alpha4-release-note.md [hunk 3]
```diff
@@ -66,7 +66,61 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+
+Command-line arguments are supported for configurable variables with boolean, int, float, decimal, string, and XML types.
+
+Example usages:
+
+```ballerina
+configurable int port = ?;
+```
+
+If the configurable variable is defined in the default module or if we are using a single bal file.
```
Comment (HindujaB) [inline release-notes/swan-lake-alpha4-release-note.md:91]:
```suggestion
If the configurable variable is defined in the default module or if we are using a single `.bal` file.
```
or 
```suggestion
If the configurable variable is defined in the default module or if we are using a single Ballerina file..
```
@praneesha WDYT?

File: release-notes/swan-lake-alpha4-release-note.md [hunk 4]
```diff
@@ -66,7 +66,61 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+Configurable values can be provided with the built-in command-line option `-C`.
+
+```
+-Ckey=value
+```
+
+Key syntax:
+
+```
+key:= [[orgName .] packageName.moduleName .] variable
```
Comment (warunalakshitha) [inline release-notes/swan-lake-alpha4-release-note.md:80]: Is the correct wording should be packageName.moduleName just moduleName

File: release-notes/swan-lake-alpha4-release-note.md [hunk 5]
```diff
@@ -66,7 +66,61 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
...
+bal run -- -CorgName.packageName.moduleName.port=9090 
+java -jar executable.jar -CorgName.packageName.moduleName.port=9090 
+```
+
+##### Support for locating multiple toml files
+
+Configurable values can be provided in multiple TOML files using the `BAL_CONFIG_FILES` environment variable.
+
+The file locations can be specified in the environment variable using an OS-specific separator. The precedence order
+ to configure the values will be as specified in the environment variable. If such an environment variable is not specified, the file located in the current directory with the file name `Config.toml` will be used.
```
Comment (manuranga) [inline release-notes/swan-lake-alpha4-release-note.md:118]:
```suggestion
The file locations can be specified in the environment variable using an OS-specific separator. The precedence order will be determined by the order in which the files are specified in the environment variable. If such an environment variable is not specified, the file located in the current directory with the file name `Config.toml` will be used.
```

**Pull request merged**